pom.xml

```diff
@@ -104,37 +94,57 @@
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock-standalone</artifactId>
-      <version>2.12.0</version>
+      <version>2.22.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.9.0</version>
+      <version>2.24.5</version>
```

Comment:
Will not be testable with Java 11, needs to take version from recent Plugin POM

Reply:
fixed